server/src/classes/crew.js

```diff
@@ -62,6 +66,7 @@ export default class Crew {
   constructor(params) {
     this.id = params.id || uuid.v4();
     this.class = "Crew";
+    // TODO: Are these really the default params we want?
```

Comment:
Yes. There have to be some default params, and I want it to be pretty obvious that they are default. Also, the client code is set up so it is unlikely that the default params will ever be used in this way.

```suggestion
```